### PR TITLE
Fixed 10144: Prevalue alias not set

### PR DESCRIPTION
**Pull request by patrickdemooij9**
If there's an existing issue for this PR then this fixes https://github.com/umbraco/Umbraco-CMS/issues/10144

Can be tested by:
When adding a property to a ContentType, go into the DataType's configuration editor (e.g. the infinite editing mode). Hover your mouse cursor over a property's name and the title attribute should display the correct alias instead of a randomly generated ID as it was doing before this.

### CODE REVIEW

File: src/Umbraco.Web.UI.Client/src/views/datatypes/datatype.edit.controller.js [hunk 1]
```diff
@@ -71,6 +60,7 @@ function DataTypeEditController($scope, $routeParams, appState, navigationServic
                 vm.preValuesLoaded = true;
                 vm.content = data;
 
+                console.log(vm.content.preValues);
```
Comment (leekelleher) [inline src/Umbraco.Web.UI.Client/src/views/datatypes/datatype.edit.controller.js:63]: 👀 @patrickdemooij9 Spotted a `console.log` left in.

Comment (patrickdemooij9) [reply src/Umbraco.Web.UI.Client/src/views/datatypes/datatype.edit.controller.js]: Whoops! I've removed it

**Comment by nathanwoulfe** [comment]
Before: 
image
After:
image

Yeah, I couldn't get the screen grab with the tooltip showing. All works fine though

**Comment by nathanwoulfe** [comment]
I've merged latest v8 into the PR branch, hopefully that resolves the failing test (I don't think the test was anything related to this change, but good to get all green lights before I merge).

**Review by nathanwoulfe — APPROVED** [review]

**Comment by nul800sebastiaan** [comment]
FYI @patrickdemooij9 @nathanwoulfe 

I had to update this one a little bit, turns out just editing datatypes from the datatypes tree was working fine but when you're editing the document type and opening the datatype in infinite editing it would no longer send the `key` to the server, totally preventing us from saving the datatype because we couldn't figure out what prevalues were being sent in.

When you edit a datatype normally we see this in the collection:

image

But from infinite editing we see this, it's sending `alias` instead of `key`:

image

So I've expected either one of them now in this commit: dc27a31a6fe35a885eab9079d2e002fc53da6391 which seems to work in both scenarios now.

**Pull request merged**